Tasks/Common/docker-common/dockercommandutils.ts

```diff
@@ -91,26 +91,39 @@ export async function getLayers(connection: ContainerConnection, imageId: string
 
 function parseHistory(input: string) {
     const NOP = '#(nop)';
-    var directive = 'UNSPECIFIED';
-    var argument = '';
-    var index = input.indexOf(NOP);
+    let directive = "UNSPECIFIED";
+    let argument = "";
+    let index: number = input.indexOf(NOP);
+    const createdByMatch = "; createdBy:";
```

Comment:
are we okay with this, as this is not going to be localized.

Reply:
We are setting these same as the Docker history command arguments.  Also this data will be visible in logs only when system debug is true, So I think we need not localise